Backpack/build.gradle

```diff
@@ -44,13 +48,14 @@ dependencies {
   implementation "com.google.android.material:material:$rootProject.ext.androidMaterial"
   implementation "androidx.constraintlayout:constraintlayout:$rootProject.ext.constraintLayout"
   implementation "androidx.cardview:cardview:$rootProject.ext.cardView"
+  implementation "androidx.swiperefreshlayout:swiperefreshlayout:$rootProject.ext.swiperefreshLayout"
```

Comment:
SwipeToRefresh (we use modified progress indicator from there) was extracted to separate library.